recipes/svmbir/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "svmbir" %}
+{% set version = "0.2.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 45137732c9a345b570a03c205c37080d23def594245b62b5bc3b18b0ba5f3f6d
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy =1.21
```

Comment:
This should only be in host
```suggestion
```

Reply:
These were all added as part of attempts to resolve build errors, but I'll try them one by one to check if they're really needed.